### DISCUSSION TITLE
Welcome to supergrub Discussions!

### DISCUSSION THREAD

**Discussion by adrian15** [supergrub/supergrub#81] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JesusGod-Pope666-Info** [comment]
Lets get some activity going :)

**Reply by JesusGod-Pope666-Info** [reply]
I am Darkijah from Denmark and homeless, I Hack the Planet and usually have a Gnulix USB vvith some storage - yet I vvass forced to find a solution because my old bootable things did not vvork vvith my nevv expensive crapbar. So I found this project using the GOLDEN USB Key to get access to my system on another USB pen vvith my persistent Crunchbang++.
Although more complicated, but at least I can get access to it.

Novv looking into if I might be able to do something further vvith my ovvn HACK THE PLANET and added this thing as vvell. VVee vvill see.
